src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -53,14 +54,15 @@ export async function fetchOCIFeature(params: CommonParams, featureSet: FeatureS
 
 	const { featureRef } = featureSet.sourceInformation;
 
-	const blobUrl = `https://${featureSet.sourceInformation.featureRef.registry}/v2/${featureSet.sourceInformation.featureRef.path}/blobs/${featureSet.sourceInformation.manifest?.layers[0].digest}`;
+	const expectedDigest = featureSet.sourceInformation.manifest?.layers[0].digest;
```

Comment:
To avoid confusion in the future, I may name this variable `blobDigest` or `layerDigest` to disambiguate from the manifest's digest